tests/scripts/get_chromedriver.sh

```diff
@@ -8,10 +8,10 @@ fi
 echo "downloading chromedriver"
 
 if [[ $os_name == 'Linux' && ! -f $chromedriver_dir/chromedriver ]]; then
-  cd chromedriver  && curl -L https://chromedriver.storage.googleapis.com/2.39/chromedriver_linux64.zip > tmp.zip &&  unzip -o tmp.zip && rm tmp.zip 
+  cd chromedriver  && curl -L https://chromedriver.storage.googleapis.com/2.43/chromedriver_linux64.zip > tmp.zip &&  unzip -o tmp.zip && rm tmp.zip 
```

Comment:
The latest is now 2.44 (sorry for the delayed review).

Reply:
Updated.  I'll merge it once travis finishes.